### PR TITLE
Main nav and categories info

### PR DESCRIPTION
**Pull request by mauricio-florez**

### CODE REVIEW

File: localPackages/altitude-commercetools-connector/src/category.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export default async function category(params, req, res) {
+  return await fulfillAPIRequest(req, {
+      appData: createAppData,
+      pageData: () => getPageData(params, req),
+  })
+}
+
+async function getPageData(params, req) {
+  const lang = "en-CA"
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/category.js:13]: we could extract the languages into a single file and re use it like:
{ 
  canada: "en-CA",
  usa: "en-US"
}

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizePlp.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { type PlpResponse } from './../types/index'
+import normalizeProduct from './normalizeProduct'
+
+export function normalizePlp(search): PlpResponse {
+  // TODO: Limit needs to be a config param
+  const limit = 24
+  // TODO: Check logic in client side, it is adding an additional page
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizePlp.ts:7]: can we continue with this TODO?

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+       }}
+    }) 
+  }
+
+  function getVariationsColor() { 
+   
+   
+    if(!data.masterVariant)
+    return []
+    //let variant =ProductVarients.reduce((prev, variant) => prev || variant.attributes.find(item => item.name === 'color'), undefined)
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:68]: is this commented code useful for later?

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      src:mastervarient.value,
+      alt:mastervarient.name,
+      magnify:{ 
+       height: 1200,
+       width: 800,
+       src: mastervarient.value
+     }
+    }
+  }
+  // const colors: {} = getVariationsColor()
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:101]: should we remove the comment?

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  const sizes = getVariationsSize('brandSize1')
+  const media = {
+    full:[getImages2('mainImage')],
+    thumbnails: [getImages2('mainImage')]//getImages('medium'),
+  }
+
+
+  data['price']= 100;// (data.masterVariant.prices[0].value.centAmount)/100 || 100;
+  const id = data.id;// data.id || data.productId || 12321;
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:110]: should we remove the comment?

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  const specs = {}  
+  return {
+    id,
+    url: `/p/${id}`,
+    name: data.name['en-CA'],//|| data.productName,
+    price: data.price,
+    priceText: `$${data.price}.00`, // n/a
+    // rating: n/a
+    description:'Ex proident nisi laborum et sint aliquip dolor cupidatat pariatur sint reprehenderit incididunt duis. Ipsum fugiat adipisicing excepteur non id aliquip cupidatat culpa ex. Aliquip consectetur tempor voluptate nulla excepteur magna non. Enim enim sunt ea labore ex aliqua qui. Velit sint irure duis excepteur fugiat elit voluptate anim sint. Anim voluptate duis aliqua duis excepteur in labore aliqua. Deserunt dolore incididunt aliquip duis. Fugiat nulla sint esse in et nisi enim sit. Reprehenderit in non labore est sit voluptate ipsum sit. Proident culpa ex mollit minim laboris ipsum incididunt dolore quis proident esse ea.',
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:119]: that long text seems hardcoded, should we keep it?

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+    specs: 'Ut ullamco dolor dolor velit nisi consectetur. Exercitation sint eu labore dolore nostrud nostrud occaecat ex nulla. Velit culpa ex quis minim Lorem quis consequat anim excepteur. Irure anim ea dolor minim aute quis anim duis ea duis irure sit Lorem irure. Ad duis officia exercitation aute aute incididunt tempor non in sunt nisi ut. Dolor consequat et sint tempor amet irure voluptate id. Dolore non cupidatat Lorem minim. Velit consectetur id eiusmod ad ea sint voluptate tempor duis quis excepteur. Dolore sunt ullamco est incididunt sit et. Sint aute nulla excepteur exercitation velit non.',// data.longDescription,
+    media,
+    thumbnail:media.thumbnails[0],
+    colors: {},
+    sizes,    
+    quantity: data.quantity || 1,
+    // TODO: Remove later
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:126]: is now later? 😅

File: localPackages/altitude-commercetools-connector/src/clients/contentful/contentful-client.js [hunk 8]
```diff
@@ -19,7 +21,83 @@ export default function getContentFulClient(req) {
...
+                    ...baseCategory,
+                    parentId: parentCategory.sys.id
+                })
+            }
+            }
+        }
+
+        let _categoryTree = arrayToTree(categories, {dataField: null})
+
+        console.log(_categoryTree)
```
Comment (alejandrosobko) [inline localPackages/altitude-commercetools-connector/src/clients/contentful/contentful-client.js:72]: we could remove this console.log

File: localPackages/altitude-commercetools-connector/src/category.js (hunk 1 above)
Comment (mauricio-florez) [reply localPackages/altitude-commercetools-connector/src/category.js:13]: actually, I'm implementing the localization part in a new branch based on Next.js structure, so don't worry this is merely for the initial example

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizePlp.ts (hunk 2 above)
Comment (mauricio-florez) [reply localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizePlp.ts:7]: that could be addressed in your PR which is mostly focused on the PLP, does it make sense?

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js (hunk 4 above)
Comment (mauricio-florez) [reply localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:101]: good catch

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js (hunk 5 above)
Comment (mauricio-florez) [reply localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:110]: removing it...

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js (hunk 6 above)
Comment (mauricio-florez) [reply localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:119]: we can keep it, for now, once Linat implements the PDP he should consume the real description from CT.  There's another pending task to type the product response, we can address part of it there

File: localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js (hunk 7 above)
Comment (mauricio-florez) [reply localPackages/altitude-commercetools-connector/src/clients/commercetools-v2/mappers/normalizeProduct.js:126]: I meant we need to evaluate the product information we need for the UI component. AS I said when we create the product type we can get rid off unnecessary props

File: localPackages/altitude-commercetools-connector/src/clients/contentful/contentful-client.js (hunk 8 above)
Comment (mauricio-florez) [reply localPackages/altitude-commercetools-connector/src/clients/contentful/contentful-client.js:72]: 👍

**Review by alejandrosobko — APPROVED** [review]

**Pull request merged**